src/codegen.ml

```diff
@@ -176,12 +176,16 @@ let get_arity_func_type_idx arity env =
     Deque.size !(env.func_types) - 1
   | Some((i, _)) -> i
 
+(* Seems a little silly when named this way, but it makes a little more sense in the
+   context of this file *)
+let encode_string : string -> int list = Utf8.decode
```

Comment:
I don't think this is named sillyly. That's what it's doing. I don't think you need the comment. Or rather, have the comment explain why it's like this.

Reply:
You don't think
```
encode = decode
```
is silly?

Reply:
I find it stupid, not silly. But it makes sense in this context. So I'd update the comment to explain why we're doing that, since this is something that looks just plain wrong/a bug.